backend/app/app/api/api_v1/endpoints/study.py

```diff
@@ -59,6 +59,13 @@ def get_next_set(
         user = current_user
     study_set = crud.studyset.get_study_set(db, user=user, deck_ids=deck_ids, return_limit=limit, force_new=force_new, is_resume=is_resume)
     print("\n\nSTUDY SET:", study_set.facts, len(study_set.facts), '\n\n')
+
+    for i in range(len(study_set.facts)):
```

Comment:
This is necessary because extra isn't automatically serialized?

Reply:
Fixed